lib.py

```diff
@@ -1,8 +1,18 @@
 # read an input file and return a list of tuples where each tuple is an (x, y) pair
 def read_inputs(filename):
-    return 0
+    with open(filename) as f:
+        contents = f.read()
+    contents_list = contents.strip().splitlines()
+    points_list = list()
+    for pair in contents_list:
+        coords = [int(coord) for coord in pair.split(' ')]
```

Comment:
This is returning a list of lists. I think this could be changed to
```python
coords = tuple(int(coord) for coord in pair.split(' '))
```
I changed this locally and it seemed to work.